client/src/app/kommissions/view/kommission-view.component.ts

```diff
@@ -25,7 +25,7 @@ export class KommissionViewComponent implements OnInit {
   }
 
   getBarmen(active: boolean): Barman[] {
-    if (this.kommission) return this.kommission.barmen.filter(b => b.active === active);
+    if (this.kommission) return this.kommission.barmen.filter(b => active ? b.isActive() : !b.isActive());
```

Comment:
Maybe use something like 
```suggestion
    if (this.kommission) return this.kommission.barmen.filter(b => b.isActive() === active);
```

Reply:
Nice one, but the correct answer is:
`if (this.kommission) return this.kommission.barmen.filter(b => (new Barman(b).isActive()) === active);`